Kernel/Memory/RegionTree.cpp

```diff
@@ -121,6 +121,23 @@ ErrorOr<VirtualRange> RegionTree::allocate_range_randomized(size_t size, size_t
         if (!m_total_range.contains(random_address, size))
             continue;
 
+#if ARCH(I386)
+        VirtualAddress random_address_end { random_address.get() + size };
+        constexpr size_t limiter_minimum_allocations = 200;
+        constexpr size_t limiter_maximum_space = 1 * MiB;
```

Comment:
What's a "limiter" in this context?

Reply:
Nothing in particular. I renamed the variables to hopefully make it more clear.